Gruntfile.js

```diff
@@ -0,0 +1,352 @@
+module.exports = function(grunt) {
+
+    var fs = require('fs');
+
+    var awsConfig = {
+        accessKey: false,
+        secretKey: false,
+        region: false,
+        s3Bucket: false
+    };
+
+    // Breaks out the keys and also gives us a nice reference of what
+    // needs to be in here without being able to be changed.
+    var awsConfigKeys = Object.keys(awsConfig);
+
+    // Take inputs for the awsConfig, prefixed with 'aws.'.
+    // For example accessKey can be inputted with '--aws.accessKey=accessKey'
+    awsConfigKeys.forEach(function(key) {
+        var optionKey = 'aws.' + key;
+        var optionValue = grunt.option(optionKey);
+        if (optionValue) {
+            // Use the original key to update it.
+            awsConfig[key] = optionValue;
+        }
+    });
+
+    grunt.file.move = function(from,to) {
+        grunt.log.writeln('Moved '+from.cyan+' to '+to.cyan);
+        grunt.file.copy(from,to);
+        grunt.file.delete(from);
+    };
+
+    grunt.loadNpmTasks('grunt-contrib-compress');
+    grunt.loadNpmTasks('grunt-contrib-requirejs');
+    grunt.loadNpmTasks('grunt-contrib-watch');
+    grunt.loadNpmTasks('grunt-version');
+    grunt.loadNpmTasks('grunt-aws');
+    grunt.loadNpmTasks('grunt-fastly');
+    grunt.loadNpmTasks('grunt-git');
+
+    grunt.initConfig({
+        aws: awsConfig,
+        version: {
+            options: {
+                pkg: grunt.file.readJSON('package.json')
+            },
+            header: {
+                options: {
+                    prefix: '@version *'
+                },
+                src: ['dist/plugins/**/*.js']
+            },
+            constant: {
+                options: {
+                    prefix: 'VERSION\\s*:\\s*[\'"]'
+                },
+                src: ['dist/plugins/**/*.js']
+            }
```

Comment:
Thinking out loud here. This task is going to take all the files in `dist/plugins/**/*.js` and regex replace the version number in them. I'm looking at these regexes and I don't see them in the eloqua file. Is that something requirejs is going to add?